src/main/java/com/bkravets/utilities/StreamUtilities.java

```diff
@@ -0,0 +1,37 @@
+package com.bkravets.utilities;
+
+import com.bkravets.models.Person;
+import com.bkravets.models.Sex;
+
+import java.util.List;
+
+public class StreamUtilities {
+
+    public static List<Person> getWomenByAgeBetweenEighteenAndSixty(List<Person> people) {
+        return people.stream()
+                .filter(person -> person.sex().equals(Sex.WOMAN))
+                .filter(person -> person.age() >= 18 && person.age() < 60)
+                .toList();
+    }
+
+    public static double getAverageAgeAmongMen(List<Person> people) {
+        return people.stream()
+                .filter(person -> person.sex().equals(Sex.MAN))
+                .mapToInt(Person::age)
+                .average()
+                .orElseThrow(() -> new IllegalArgumentException("List cannot be empty"));
```

Comment:
It's not necessary that list is empty. Could be that there are only women in the list :)